### PR TITLE
Replace /var/lib/pgsql/ with /var/opt/rh/rh-postgresql12/lib/pgsql/

### PR DESCRIPTION
**Pull request by ryandeussing**
This change exists in master and in 2.4, but apparently did not make it to 2.3 or 2.1

Bug 1946985 - Update references from /var/lib/pgsql to /var/opt/rh/rh-postgresql12/lib/pgsql

https://bugzilla.redhat.com/show_bug.cgi?id=1946985

Cherry-pick into:

* [ ] Foreman 2.5 (Satellite 6.10)
* [ ] Foreman 2.4
* [ ] Foreman 2.3 (Satellite 6.9)
* [x] Foreman 2.1 (Satellite 6.8)

### CODE REVIEW

**Review by melcorr — APPROVED** [review]
Thanks! 
Eric had updated this in master as part of the work he was doing to prepare 2.4.

**Comment by melcorr** [comment]
This is flagged as draft, so will wait to merge incase there are additions!

File: guides/doc-Administering_Red_Hat_Satellite/topics/proc_estimating-the-size-of-a-backup.adoc [hunk 1]
```diff
@@ -20,9 +20,9 @@ A full backup requires space to store the following data:
 . Enter the `du` command to estimate the size of uncompressed directories containing {Project} database and configuration files:
 +
 ----
-# du -sh /var/lib/mongodb /var/lib/pgsql/data /var/lib/pulp
+# du -sh /var/lib/mongodb /var/opt/rh/rh-postgresql12/lib/pgsqldata /var/lib/pulp
```
Comment (ekohl) [inline guides/doc-Administering_Red_Hat_Satellite/topics/proc_estimating-the-size-of-a-backup.adoc:23]: This doesn't looks correct (repeated multiple times)
```suggestion
# du -sh /var/lib/mongodb /var/opt/rh/rh-postgresql12/lib/pgsql/data /var/lib/pulp
```

**Review by ekohl — CHANGES_REQUESTED** [review]

File: guides/doc-Administering_Red_Hat_Satellite/topics/proc_estimating-the-size-of-a-backup.adoc [hunk 2]
```diff
@@ -20,9 +20,9 @@ A full backup requires space to store the following data:
 . Enter the `du` command to estimate the size of uncompressed directories containing {Project} database and configuration files:
 +
 ----
-# du -sh /var/lib/mongodb /var/lib/pgsql/data /var/lib/pulp
+# du -sh /var/lib/mongodb /var/opt/rh/rh-postgresql12/lib/pgsql/data /var/lib/pulp
 480G	/var/lib/mongodb
-100G    /var/lib/pgsql/data
+100G    /var/opt/rh/rh-postgresql12/lib/pgsqldata
```
Comment (ekohl) [inline guides/doc-Administering_Red_Hat_Satellite/topics/proc_estimating-the-size-of-a-backup.adoc:25]: Please fix all occurrences :)
```suggestion
100G    /var/opt/rh/rh-postgresql12/lib/pgsql/data
```

**Review by ekohl — CHANGES_REQUESTED** [review]

File: guides/doc-Administering_Red_Hat_Satellite/topics/proc_performing-a-snapshot-backup.adoc [hunk 3]
```diff
@@ -23,7 +23,7 @@ Ensure no other tasks are scheduled for the same time as the backup.
 
 Before you perform the snapshot backup, ensure that the following conditions are met:
 
-* The system uses LVM for the directories that you snapshot: `/var/lib/pulp/`, `/var/lib/mongodb/`, and `/var/lib/pgsql/`.
+* The system uses LVM for the directories that you snapshot: `/var/lib/pulp/`, `/var/lib/mongodb/`, and `/var/opt/rh/rh-postgresql12/lib/pgsql`.
```
Comment (maximiliankolb) [inline guides/doc-Administering_Red_Hat_Satellite/topics/proc_performing-a-snapshot-backup.adoc:26]:
```suggestion
* The system uses LVM for the directories that you snapshot: `/var/lib/pulp/`, `/var/lib/mongodb/`, and `/var/opt/rh/rh-postgresql12/lib/pgsql/`.
```

**Review by maximiliankolb — CHANGES_REQUESTED** [review]
apart from the open threads, everythings LGTM.

**Review by ekohl — APPROVED** [review]

**Pull request merged**